ext/CEGUI/CEGUI.cpp

```diff
@@ -97,6 +98,15 @@ void GUIManager::init()
     ));
 }
 
+void GUIManager::render()
+{
+    auto renderState = irr::video::COpenGLState::collectGLState();
+    irr::video::COpenGLState guiState;
```

Comment:
default constructor will make an `COpenGLState` object that is as-if for a default OpenGL state that you get at startup, just pointing this out.. don't know if thats the state you want CEGUI to start out in

Reply:
Added to task list, will review this when things render.

Reply:
done in latest commit